token/token.go

```diff
@@ -18,12 +18,52 @@ const (
 
 	Minus
 	Plus
-
+	Mul
+	Rem
+	Quo
+	Inc
+	Dec
 	Dot
 	LParen
 	RParen
 	Comma
 	SemiColon
+	LBrace
+	RBrace
+	LBrack
+	RBrack
+	Less
+	Greater
+	LessEq
+	GreaterEq
+	Equal
+	NotEqual
+	TEqual // stands for triple equal x_x
+	NotTEqual
+	LShift
+	RShift
+	RShiftZero // Right Shift with zero filling, yay JS
+	And
+	Or
+	Xor
+	Not
+	LNot
```

Comment:
LNot ?

Reply:
Logical Not maybe (!).
`Not` probably means the bitwise Not (~).

Reply:
Indeed logical not, I copied the names from Go's scanner as far as I can remember =P